app/models/library_entry.rb

```diff
@@ -241,60 +239,15 @@ def kind
   end
 
   after_create do
-    # Activity History Stat will be using this
-    library_event = LibraryEvent.create_for(:added, self)
-    # Stat STI
-    case kind
-    when :anime
-      Stat::AnimeCategoryBreakdown.increment(user, self)
-      Stat::AnimeAmountConsumed.increment(user, self)
-      Stat::AnimeFavoriteYear.increment(user, self)
-      Stat::AnimeActivityHistory.increment(user, library_event)
-    when :manga
-      Stat::MangaCategoryBreakdown.increment(user, self)
-      Stat::MangaAmountConsumed.increment(user, self)
-      Stat::MangaFavoriteYear.increment(user, self)
-      Stat::MangaActivityHistory.increment(user, library_event)
-    end
+    Stat::AddWorker.perform_async(kind, user, id)
```

Comment:
We may add non-library related stats, so this should be named something like `Stat::LibraryCreateWorker`

Reply:
yup, will rename all 3 of them.